### PR TITLE
Add time field converter

### PR DESCRIPTION
**Pull request by ischaojie**
Looks like there is no time field converter :)

### CODE REVIEW

**Comment by ischaojie** [comment]
Fix the lint :)

**Comment by ischaojie** [comment]
I'm not sure what this means, `cadecov/patch` failed.

**Comment by aminalaee** [comment]
There's no test for this, it's missing coverage. There's a model which has all the fields for coverage, you can add it there.

**Comment by ischaojie** [comment]
> There's no test for this, it's missing coverage. There's a model which has all the fields for coverage, you can add it there.

Got it!

**Comment by ischaojie** [comment]
Add a test for this:)

File: tests/test_forms.py [hunk 1]
```diff
@@ -243,3 +244,14 @@ class CustomModel(Base):
 async def test_model_form_include_pk() -> None:
     Form = await get_model_form(model=User, engine=engine, form_include_pk=True)
     assert "id" in Form()._fields
+
+
+async def test_form_time_converter() -> None:
+    class ExampleModel(Base):
+        __tablename__ = "example_model"
+        id = Column(Integer, primary_key=True)
+        start_time = Column(Time)
```
Comment (aminalaee) [inline tests/test_forms.py:253]: You can add this new field to here, I think it won't need a separate test.

Comment (aminalaee) [reply tests/test_forms.py:253]: I mean if you add this Time field to the `User` class in there, you can just assert it in `test_model_form` and no need for the new test.

Comment (ischaojie) [reply tests/test_forms.py:253]: got it.

**Pull request merged**